ooniapi/services/oonirun/src/oonirun/routers/v2.py

```diff
@@ -151,9 +171,9 @@ class OONIRunLinkCreateEdit(OONIRunLinkBase):
 )
 def create_oonirun_link(
     create_request: OONIRunLinkCreateEdit,
+    db: PostgresSession,
```

Comment:
This is an interesting way to defined the dependencies I wasn't aware of: https://fastapi.tiangolo.com/tutorial/dependencies/#declare-the-dependency-in-the-dependant.

I wonder if at some point we should refactor the other dependencies to use it too.